fe2-android/app/build.gradle

```
@@ -16,179 +16,147 @@ android {
 
         testInstrumentationRunner "com.github.dedis.popstellar.HiltTestRunner"
 
+        // The following feature is not working properly with the coverage on the current build version
+        // As it's purpose is to clear physical data between each tests, its is not used yet.
+        // Let's hope the feature will be fixed before we actually need it.
+
```

I would add an empty comment here, to signal that it's a continuous comment.
```suggestion
        // 
```